src/libs/actions/BankAccounts.ts

```diff
@@ -57,18 +61,16 @@ function clearOnfidoToken() {
 
 /**
  * Helper method to build the Onyx data required during setup of a Verified Business Bank Account
- *
- * @returns {Object}
  */
-function getVBBADataForOnyx() {
+function getVBBADataForOnyx(): OnyxData {
     return {
         optimisticData: [
             {
                 onyxMethod: Onyx.METHOD.MERGE,
                 key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
                 value: {
                     isLoading: true,
-                    errors: null,
+                    errors: undefined,
```

Comment:
AFAIK, to clear value from **Onyx** we use `null`. Why do we need this change here?

Reply:
@rezkiy37 with null it gives this type error
![image](https://github.com/Expensify/App/assets/23176449/4135970e-3a16-4ff0-8106-71c595d0d2ad)

Reply:
Sounds not good.